### PR TITLE
[ci skip] Add link to testing guide

### PR DESCRIPTION
**Pull request by brooksreese**
This guide does not give an in-depth look at testing, but there is a brief overview.  Because of this, I gave a link to the testing guide and told the user that we would briefly touch on testing in section 2.7.

### CODE REVIEW

File: guides/source/command_line.md [hunk 1]
```diff
@@ -260,7 +260,13 @@ $ bin/rake db:migrate
 ==  CreateHighScores: migrated (0.0019s) ======================================
 ```
 
-INFO: Let's talk about unit tests. Unit tests are code that tests and makes assertions about code. In unit testing, we take a little part of code, say a method of a model, and test its inputs and outputs. Unit tests are your friend. The sooner you make peace with the fact that your quality of life will drastically increase when you unit test your code, the better. Seriously. We'll make one in a moment.
+INFO: Let's talk about unit tests. Unit tests are code that tests and makes assertions 
+about code. In unit testing, we take a little part of code, say a method of a model, 
+and test its inputs and outputs. Unit tests are your friend. The sooner you make 
+peace with the fact that your quality of life will drastically increase when you unit 
+test your code, the better. Seriously. We will touch on testing in section 2.7, but
```
Comment (rafaelfranca) [inline guides/source/command_line.md]: We should not link the section number since it can change. Can we keep the original text?

Comment (brooksreese) [inline guides/source/command_line.md]: > Can we keep the original text?

Are you suggesting to revert the commit or remove the section number but keep a link to the testing guide?  I see your point about section numbers changing.

Comment (rafaelfranca) [inline guides/source/command_line.md]: Just remove the section number keeping the link.

**Comment by brooksreese** [comment]
@rafaelfranca I pushed the changes we discussed, please take a look and let me know what you think.

**Pull request merged**